libraries/AP_NavEKF2/AP_NavEKF2_MagFusion.cpp

```diff
@@ -388,6 +389,17 @@ void NavEKF2_core::FuseMagnetometer()
         MagPred[1] = DCM[1][0]*magN + DCM[1][1]*magE  + DCM[1][2]*magD + magYbias;
         MagPred[2] = DCM[2][0]*magN + DCM[2][1]*magE  + DCM[2][2]*magD + magZbias;
 
+        if (pending_z_mag_correction && frontend->_mag_learn_limit > 0) {
+            pending_z_mag_correction = false;
+            float correction = magDataDelayed.mag[2] - MagPred[2];
```

Comment:
Same comments as: https://github.com/matternet/ardupilot/pull/120

Lets define '2' with a const or constexpr etc.

Reply:
no, the code change follows the coding style for ardupilot. We do not want the matternet EKF2 code to diverge from ardupilot code